pkg/ccl/sqlproxyccl/balancer/conn_tracker.go

```diff
@@ -49,7 +54,7 @@ func NewConnTracker(
 		timeSource = timeutil.DefaultTimeSource{}
 	}
 
-	t := &ConnTracker{timeSource: timeSource}
+	t := &ConnTracker{timeSource: timeSource, verboseLogging: log.V(2)}
```

Comment:
What is log level 2 and how did you pick it? Are there corresponding constants like error/warn/info/etc.?

Reply:
Unfortunately, I don't think we have any rule of thumb around vmodule logging levels. See this internal Slack thread: https://cockroachlabs.slack.com/archives/C9TGBJB44/p1614100926026700. Each vmodule logging level seems to be specific to the file itself. I picked 2 somewhat arbitrarily; it's a middle value, and opens up opportunities for something less/more verbose as well.

For this case, if we wanted to display logs from the connection tracker, we would start the proxy with the following flag: `--vmodule=conn_tracker=2` (i.e. `--vmodule=FILE=LEVEL,FILE2=LEVEL2,...`).